src/System.Memory/src/System/Buffers/ReadOnlyBufferExtensions.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma warning disable 1591
+
+namespace System.Buffers
+{
+    public static class ReadOnlyBufferExtensions
+    {
+        public static SequencePosition? PositionOf<T>(this ReadOnlyBuffer<T> buffer, T value) where T: IEquatable<T>
+        {
+            SequencePosition position = buffer.Start;
+            SequencePosition result = position;
+            while (buffer.TryGet(ref position, out var memory))
+            {
+                var index = memory.Span.IndexOf(value);
+                if (index != -1)
+                {
+                    return buffer.GetPosition(result, index);
+                }
+                result = position;
+            }
+            return null;
+        }
+
+
+        /// <summary>
+        /// Copy the <see cref="ReadOnlyBuffer{T}"/> to the specified <see cref="Span{Byte}"/>.
+        /// </summary>
+        /// <param name="buffer">The source <see cref="ReadOnlyBuffer{T}"/>.</param>
```

Comment:
These xml comments do not add much value and can be removed, imo (or re-written).